docs/_docs/themes.md

```diff
@@ -35,13 +48,78 @@ Jekyll will look first to your site's content, before looking to the theme's def
 Refer to your selected theme's documentation and source repository for more information on what files you can override.
 {: .note .info}
 
-To locate theme's files on your computer, run `bundle show` followed by
-the name of the theme's gem, e.g. `bundle show minima` for default Jekyll's
-theme. Then copy the files you want to override, from the returned path to your root folder.
+To locate theme's files on your computer:
+
+1.  Run `bundle show` followed by the name of the theme's gem, e.g., `bundle show minima` for default Jekyll's theme.
+
+    The location of the theme gem is returned. For example, minima is located in `/usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0` on a Mac.
+
+2.  Change to the directory's location and open the directory in Finder or Explorer:
+
+    ```
+    cd /usr/local/lib/ruby/gems/2.3.0/gems/minima-2.1.0
+    open .
+    # for Windows, use "explorer ."
+    ```
+
+    A Finder or Explorer window opens showing the theme's files and directories.
+
+    With a clear understanding of the theme's files, you can now override any theme file by creating a similarly named file in your Jekyll site directory.
+
+## Converting gem-based themes to regular themes
+
+Suppose you want to get rid of the gem-based theme and convert it to a regular theme, where all files are present in your Jekyll site directory, with nothing stored in the theme gem.
+
+To do this, copy the files from the theme gem's directory into your Jekyll site directory. (For example, copy them to `/myblog` if you created your Jekyll site at `/myblog`. See the previous section for details.)
+
+Then modify the Gemfile and configuration to remove references to the theme gem. For example, to remove Minima:
+*  Open `Gemfile` and remove `gem "minima", "~> 2.0"`.
+*  Open `_config.yml` and remove `theme: minima`.
+
+Now `bundle update` will no longer get updates for the theme gem.
+
+## Installing a gem-based theme {#installing-a-theme}
+
+The `jekyll new <PATH>` command isn't the only way to create a new Jekyll site with a gem-based theme. You can also find gem-based themes online and incorporate them into your Jekyll project.
+
+For example, search for [jekyll-theme on RubyGems](https://rubygems.org/search?utf8=%E2%9C%93&query=jekyll-theme) to find other gem-based themes. (Note that not all themes are using `jekyll-theme` as a convention in the theme name.)
 
-## Creating a theme
+To install a gem-based theme:
 
-Jekyll themes are distributed as Ruby gems. Don't worry, Jekyll will help you scaffold a new theme with the `new-theme` command. Just run `jekyll new-theme` with the theme name as an argument:
+1.  Add the theme to your site's `Gemfile`:
+
+    ```sh
+    gem 'my-awesome-jekyll-theme'
+    ```
+
+2.  Install the theme:
+
+    ```sh
+    bundle install
+    ```
+
+3. Add the following to your site's `_config.yml` to activate the theme:
```

Comment:
Missing space at the beginning of the third list here.

![code-rendering](https://cloud.githubusercontent.com/assets/103008/21519588/b6ac970c-ccee-11e6-97bc-60f0cd60e3ce.png)